content/src/service/snapshots/SnapshotManager.ts

```diff
@@ -103,6 +126,90 @@ export class SnapshotManager {
+        task,
+        async (db) => {
+          // Get all the active entities
+          const snapshot: FullSnapshot[] = await db.deployments.getFullSnapshot()
+
+          // Calculate the local deployment timestamp of the newest entity in the snapshot
+          const snapshotTimestamp = snapshot[snapshot.length - 1]?.localTimestamp ?? 0
+
+          // Format the snapshot in a tmp file
+          // const tmpFile = await createTempFile('snapshot')
+          fs.mkdirSync(path.resolve(this.contentStorageFolder, 'tmp-snapshot'), { recursive: true })
+          const tmpFile = path.resolve(this.contentStorageFolder, 'tmp-snapshot/' + Math.random())
+
+          console.log(`Name of file: ${tmpFile}`)
+
+          const writeStream = fs.createWriteStream(tmpFile)
+          const fileClosedFuture = new Promise<void>((resolve, reject) => {
+            writeStream.on('finish', resolve)
+            writeStream.on('error', reject)
+          })
+
+          console.log(`File written`)
+
+          try {
+            for (const snapshotElem of snapshot) {
+              writeStream.write(JSON.stringify(snapshotElem) + '\n')
+            }
+          } finally {
+            writeStream.close()
+            await fileClosedFuture
+          }
+
+          console.log(`Stream closed`)
+          const contentFromFile = await fs.promises.readFile(tmpFile)
+          console.log(`Could read file`)
+
+          // TODO: use require('@dcl/snapshots-fetcher/dist/utils').hashStreamV1 and UintArray instead of Buffer
+          const hash = await Hashing.calculateIPFSHash(contentFromFile)
+
+          console.log(`Hash of the file: ${hash}`)
+
+          // if success move the file to the contents folder
+          const destinationFilename = path.resolve(this.contentStorageFolder, 'contents/', hash)
+          console.log(`Moving to: ${destinationFilename}`)
+
+          // delete target file if exists
+          if (await checkFileExists(destinationFilename)) {
+            await fs.promises.unlink(destinationFilename)
+          }
+
+          console.log(`Renaming file from: ${tmpFile} to: ${destinationFilename}`)
+          // move downloaded file to target folder
+          await fs.promises.rename(tmpFile, destinationFilename)
+
+          // Store the metadata
+          this.lastSnapshotsForAllEntityTypes = { hash, lastIncludedDeploymentTimestamp: snapshotTimestamp }
+
+          // Log
+          SnapshotManager.LOGGER.debug(
+            `Generated snapshot for all entity types. It includes ${snapshot.length} active deployments. Last timestamp is ${snapshotTimestamp}`
+          )
+        },
+        { priority: DB_REQUEST_PRIORITY.HIGH }
+      )
+
+      // Delete the previous full snapshot (if it exists)
+      if (!!previousHash && this.shouldPrunePreviousSnapshot(previousHash)) {
+        await this.service.deleteContent([previousHash])
+      }
+    } catch {
+      SnapshotManager.LOGGER.debug('There was an error generating snapshot')
```

Comment:
```suggestion
    } catch (err: any) {
      SnapshotManager.LOGGER.debug('There was an error generating snapshot')
      SnapshotManager.LOGGER.error(err)
```

We will love to have those logs if something goes wrong